Libs/MRML/Core/vtkMRMLAbstractViewNode.cxx

```diff
@@ -36,26 +36,26 @@ static const char* DEFAULT_AXIS_LABELS[vtkMRMLAbstractViewNode::AxisLabelsCount]
 
 //----------------------------------------------------------------------------
 vtkMRMLAbstractViewNode::vtkMRMLAbstractViewNode()
+: LayoutLabel(NULL)
+, ViewGroup(0)
+, Active(0)
+, Visibility(1)
+, OrientationMarkerEnabled(false)
+, OrientationMarkerType(OrientationMarkerTypeNone)
+, OrientationMarkerSize(OrientationMarkerSizeMedium)
+, RulerEnabled(false)
+, RulerType(RulerTypeNone)
 {
-  this->LayoutLabel = NULL;
-  this->ViewGroup = 0;
-  this->Active = 0;
-  this->Visibility = 1;
-
-  double black[3] = {0.,0.,0.};
-  memcpy(this->BackgroundColor, black, 3 * sizeof(double));
-  memcpy(this->BackgroundColor2, black, 3 * sizeof(double));
+  this->BackgroundColor[0] = 0.0;
+  this->BackgroundColor[1] = 0.0;
+  this->BackgroundColor[2] = 0.0;
+  this->BackgroundColor2[0] = 0.0;
+  this->BackgroundColor2[1] = 0.0;
+  this->BackgroundColor2[2] = 0.0;
```

Comment:
These could also go in the initializer list as `BackgroundColor{}, BackgroundColor2{}`

Reply:
Good point. Earlier VS versions did not allow this, but this works now in VS2015:

<pre>
vtkMRMLAbstractViewNode::vtkMRMLAbstractViewNode()
  : LayoutLabel(NULL) 
  ...
  , Color{0.0, 0.0, 0.0}
{
...
</pre>

Do you know if this works on MacOSX and Linux compilers that Slicer-4.9 uses?

Reply:
I checked clang on mac sdk 10.12 already -- works fine. I'm not sure if the linux compiler version is finalized yet, but this *should* be ok in any C++11 compliant compiler.

Reply:
OK, I'll update the initialization then.

Reply:
Waiting Linux is also migrated. Could you hold on c++11 only feature ?